src/System.Runtime.Extensions/src/System/Net/WebUtility.cs

```diff
@@ -5,13 +5,15 @@
 // Don't entity encode high chars (160 to 256)
 #define ENTITY_ENCODE_HIGH_ASCII_CHARS
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
 using System.IO;
 using System.Runtime.CompilerServices;
 using System.Text;
+using System.Buffers;
```

Comment:
nit: sort in alphabetical order

Reply:
I think this import isn't used any more actually. Good catch, thanks!